src/ChatPage.cpp

```diff
@@ -691,13 +691,20 @@ ChatPage::sendNotifications(const mtx::responses::Notifications &res)
                                           this,
                                           [this, room_id, event_id, item, user_id, info](
                                             QPixmap image) {
+                                                  bool isEmote = false;
+                                                  auto ev      = cache::client()->getEvent(
+                                                    room_id.toStdString(), event_id);
+                                                  if (ev && mtx::accessors::msg_type(ev->data) ==
+                                                              mtx::events::MessageType::Emote)
+                                                          isEmote = true;
```

Comment:
```suggestion
```

But you should maybe format the whole notification body here, before you need special casing for every message type... Maybe that's not possible for every platform though, in which case it may be correct to just pass the whole event to the notification.